modules/integration/src/test/scala/scala/cli/integration/PublishSetupTests.scala

```diff
@@ -89,6 +90,9 @@ class PublishSetupTests extends ScalaCliSuite {
       .toMap
   }
 
+  override def munitFlakyOK: Boolean =
+    TestUtil.isNativeCli && TestUtil.isCI && Properties.isMac && !TestUtil.isM1
```

Comment:
I have mixed feelings about this, as if we add more flaky tests in this suite for other setups, this will have to be refactored... but eh, maybe it's ok for now.
Hopefully we won't get more flaky `publish setup` tests... 😕